src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -112,6 +113,10 @@ export const JiraConfigurationGet = async (
 				isGlobalInstall: installation.repository_selection === "all"
 			}));
 
+		const handleNavigationClassName = await booleanFlag(BooleanFlags.GHE_SERVER, false, jiraHost)
```

Comment:
I've wrapped this change in our GHE global FF so current flow in prod will remain unaffected. If the FF is off (as it is currently is for everyone) we still handle navigating via the classname add-organization-link.